discovery/watcher.go

```diff
@@ -72,7 +72,8 @@ type Watcher struct {
 
 	acls *ACLs
 
-	subcribers []chan State
+	subcribers        []chan State
```

Comment:
(nit) I just noticed that `subcribers` is misspelled.

Reply:
haha, me too, I'll fix it